### PR TITLE
Minor fixes for CFME deployment

### PR DESCRIPTION
**Pull request by machacekondra**
This PR fixes following issues:

 - Wait for IPv4
 - Find data domain before qcow deployment.
 - CFME example
 - -ansible_user: root added to Add host alias of appliance task

### CODE REVIEW

**Review by mwperina — APPROVED** [review]

File: examples/cfme.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+- name: oVirt CFME deployment
```
Comment (mykaul) [inline examples/cfme.yml]: oVirt  or RHV? Since when is CFME supported on oVirt?

File: examples/cfme.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  connection: local
+  gather_facts: false
+
+  vars_files:
+    # Contains encrypted `engine_password` and `metrics_password`
+    # varibale using ansible-vault
+    - passwords.yml
+
+  vars:
+    engine_fqdn: ovirt-engine.example.com
```
Comment (mykaul) [inline examples/cfme.yml:13]: Why aren't we taking those from an external, single vars file?

File: examples/cfme.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    engine_user: admin@internal
+    engine_password: 123456
```
Comment (mykaul) [inline examples/cfme.yml]: Why not in vault? We should advertise and promote best practices (especially downstream). I see it up with the inclusion of 'passwords.yml' ?

File: examples/cfme.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+    miq_vm_name: cfme_59
+    miq_qcow_url: https://cdn.example.com/cfme-rhevm-5.9.1.2-1.x86_64.qcow2
+    miq_vm_cluster: mycluster
+    miq_vm_root_password: securepassword
+    miq_vm_cloud_init:
+      host_name: "{{ miq_vm_name }}"
```
Comment (mykaul) [inline examples/cfme.yml:21]: Is ovirt-guest-agent-common installed on downstream appliance, or do we want to install it? (Is the appliance registered to RHSM even?)

File: examples/cfme.yml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    miq_vm_disks:
+      database:
+        name: "{{ miq_vm_name }}_Disk2"
```
Comment (mykaul) [inline examples/cfme.yml]: Why call it _Disk2 and not _database?

File: examples/cfme.yml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        size: 10GiB
+        interface: virtio
+        format: raw
+      log:
+        name: "{{ miq_vm_name }}_Disk3"
```
Comment (mykaul) [inline examples/cfme.yml]: Why call it _Disk3and not _log?

File: examples/cfme.yml [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+        size: 10GiB
+        interface: virtio
+        format: cow
+      tmp:
+        name: "{{ miq_vm_name }}_Disk4"
```
Comment (mykaul) [inline examples/cfme.yml]: Why call it _Disk4 and not _tmp?

File: examples/cfme.yml [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+        size: 10GiB
+        interface: virtio
+        format: cow
```
Comment (mykaul) [inline examples/cfme.yml]: Why not raw?

File: examples/cfme.yml [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+        format: raw
+      log:
+        name: "{{ miq_vm_name }}_Disk3"
+        size: 10GiB
+        interface: virtio
+        format: cow
+      tmp:
+        name: "{{ miq_vm_name }}_Disk4"
+        size: 10GiB
+        interface: virtio
```
Comment (mykaul) [inline examples/cfme.yml]: Probably should use virtio-scsi and allow discard?

**Review by mykaul — COMMENTED** [review]
Small comments within. Nothing that can't be fixed in a follow-up patch.

**Comment by mykaul** [comment]
Also noticed the VM is not configured as highly available ('high_availability: true' in the VM definition).

File: examples/cfme.yml (hunk 1 above)
Comment (machacekondra) [reply examples/cfme.yml]: I may use RHV here, but I think CFME should run nicely on oVirt as well even if it's unsupported.

File: examples/cfme.yml (hunk 2 above)
Comment (machacekondra) [reply examples/cfme.yml:13]: We can. But I think there is no real need for this.

File: examples/cfme.yml (hunk 3 above)
Comment (machacekondra) [reply examples/cfme.yml]: right, thanks, will remove.

File: examples/cfme.yml (hunk 4 above)
Comment (machacekondra) [reply examples/cfme.yml:21]: GA is installed. Not sure if it's registered.

File: examples/cfme.yml (hunk 5 above)
Comment (machacekondra) [reply examples/cfme.yml]: Yes, makes more sense, thanks.

File: examples/cfme.yml (hunk 6 above)
Comment (machacekondra) [reply examples/cfme.yml]: Yes, makes more sense, thanks.

File: examples/cfme.yml (hunk 7 above)
Comment (machacekondra) [reply examples/cfme.yml]: Yes, makes more sense, thanks.

File: examples/cfme.yml (hunk 9 above)
Comment (machacekondra) [reply examples/cfme.yml]: I can add virtio-scsi, but `ovirt_disks` module don't support discrad functionality yet.

File: examples/cfme.yml (hunk 8 above)
Comment (machacekondra) [reply examples/cfme.yml]: No strong reason, I can change.

**Pull request merged**